### PR TITLE
 deploy to prod

### PR DESCRIPTION
**Pull request by marksantoso**

### CODE REVIEW

File: content/content-strategy/removing-content/archive-record-content/archiving-comm02.md [hunk 1]
```diff
@@ -3,7 +3,6 @@ layout: text/textblock
 section: Archiving for Commonwealth records
 ---
 ## [2]If your website is not stored with the National Library
-
-If the site you’re removing is not archived with the AGWA, email them and make sure it's captured in their next website harvest: [webarchive@nla.gov.au](mailto:webarchive@nla.gov.au)
+If the site you’re removing is not archived with the AGWA, email them and make sure it's captured in their next website harvest: [recordkeeping@naa.gov.au](mailto:recordkeeping@naa.gov.au)
```
Comment (joolswood) [inline content/content-strategy/removing-content/archive-record-content/archiving-comm02.md]: This commit is wrong, should be the `master` version (`NLA`).

File: content/content-strategy/index.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 title: Content strategy
 weight: 40
+description: A content strategy guides the creation, delivery and governance of useful, usable content.
```
Comment (joolswood) [inline content/content-strategy/index.yml]: Ok

File: content/content-strategy/information-architecture/buy-in-for-ia/index.yml [hunk 3]
```diff
@@ -1,12 +1,13 @@
 title: Getting buy-in for your IA
 weight: 10
+description: Before you develop or update an IA, you may need to seek approval for additional resources.
```
Comment (joolswood) [inline content/content-strategy/information-architecture/buy-in-for-ia/index.yml]: Ok

File: content/content-strategy/information-architecture/buy-in-for-ia/index.yml [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
 layout: layout/topic
 theme: blue
 header:
   - /_shared/globalheader.md
   - /_shared/header.md
 main:
-  - intro.md          
+  - intro.md
```
Comment (joolswood) [inline content/content-strategy/information-architecture/buy-in-for-ia/index.yml]: Ok

File: content/content-strategy/information-architecture/develop-ia/index.yml [hunk 5]
```diff
@@ -1,12 +1,13 @@
 title: Develop your IA
 weight: 60
+description: Here are a range of things to think about while you are developing your IA.
```
Comment (joolswood) [inline content/content-strategy/information-architecture/develop-ia/index.yml]: Ok

File: content/content-strategy/information-architecture/develop-ia/index.yml [hunk 6]
```diff
@@ -1,12 +1,13 @@
...
 layout: layout/topic
 theme: blue
 header:
   - /_shared/globalheader.md
   - /_shared/header.md
 main:
-  - intro.md          
+  - intro.md
```
Comment (joolswood) [inline content/content-strategy/information-architecture/develop-ia/index.yml]: Ok

File: content/content-strategy/information-architecture/engage-about-ia/index.yml [hunk 7]
```diff
@@ -1,13 +1,14 @@
 title: Engage about your IA
 weight: 40
+description: Work with other government agencies and engage with users early in your IA planning.
```
Comment (joolswood) [inline content/content-strategy/information-architecture/engage-about-ia/index.yml]: Ok

File: content/content-strategy/information-architecture/ia-tools-techniques/index.yml [hunk 8]
```diff
@@ -1,18 +1,19 @@
 title: IA tools and techniques
 weight: 90
+description: Here are some tools and techniques which can help when developing your IA.
```
Comment (shad0wMe0w) [inline content/content-strategy/information-architecture/ia-tools-techniques/index.yml]: Ok

File: content/content-strategy/information-architecture/engage-about-ia/index.yml [hunk 9]
```diff
@@ -1,13 +1,14 @@
...
 layout: layout/topic
 theme: blue
 header:
   - /_shared/globalheader.md
   - /_shared/header.md
 main:
   - intro.md
-  - sections.md          
+  - sections.md
```
Comment (joolswood) [inline content/content-strategy/information-architecture/engage-about-ia/index.yml]: Ok

File: content/content-strategy/information-architecture/ia-tools-techniques/index.yml (hunk 8 above)
Comment (joolswood) [inline content/content-strategy/information-architecture/ia-tools-techniques/index.yml]: Ok

File: content/content-strategy/information-architecture/ia-tools-techniques/index.yml [hunk 10]
```diff
@@ -1,18 +1,19 @@
...
 layout: layout/topic
 theme: blue
 header:
   - /_shared/globalheader.md
   - /_shared/header.md
 main:
-  - intro.md          
+  - intro.md
```
Comment (joolswood) [inline content/content-strategy/information-architecture/ia-tools-techniques/index.yml]: Ok

File: content/governing-content/content-lifecycle/planning-content/activities.md [hunk 11]
```diff
@@ -13,7 +13,3 @@ Once your plan is complete, use it inform everyone involved.
 Use this workshop activity with your immediate or broader teams. Explore the tasks involved in each stage of the content lifecycle and the skills you need to execute those tasks.
 
 Invite all of the people who are involved in planning and creating, through to removing content.
-
```
Comment (joolswood) [inline content/governing-content/content-lifecycle/planning-content/activities.md]: Should be ok but double check with Matt.

File: content/user-research/analysing-user-research/affinity.md [hunk 12]
```diff
@@ -9,4 +9,4 @@ reverse: false
 imageAlt: "A wall with a lot of post-it notes in different colours."
 ---
 
-‘We did affinity mapping in Alpha; other teams do it earlier in Discovery. It’s what we did after we spoke to the users. We used it to gather the user insights to find high-level topics and headings. Those fed into design hypotheses. Going analogue like this gave us good visibility. It’s hands on. If we did it digitally straight away the team might not have had the level of comfort to all get involved.’ — Delivery manager
+‘We did affinity mapping in Alpha; other teams do it earlier in Discovery. It’s what we did after we spoke to the users. We used it to gather the user insights to find high-level topics and headings. Those fed into design hypotheses. Going analogue like this gave us good visibility. It’s hands on. If we did it digitally straightaway the team might not have had the level of comfort to all get involved.’ — Delivery manager
```
Comment (joolswood) [inline content/user-research/analysing-user-research/affinity.md]: Ok

File: content/governing-content/content-lifecycle/planning-content/activities.md (hunk 11 above)
Comment (shad0wMe0w) [inline content/governing-content/content-lifecycle/planning-content/activities.md]: This block is still on Prod
- AB

File: content/user-research/analysing-user-research/decide.md [hunk 13]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: text/textblock
 ---
-You can use your findings to make decisions about what to work on, change or research next. This supports the agile method of planning continuously based on new facts or requirements.
+You can use your findings to make decisions about what to work on, change or research next. This supports the agile method of continuous planning with new facts or requirements.
```
Comment (joolswood) [inline content/user-research/analysing-user-research/decide.md]: ok

File: content/user-research/analysing-user-research/decide.md [hunk 14]
```diff
@@ -1,9 +1,9 @@
...
 
-As a group, discuss if there are any obvious actions you want to take. Write these on a third colour sticky note and add it to the relevant group on your affinity map. Actions might include:
+As a group, discuss if there are any actions you want to take. Write these on sticky notes in another colour. Add them to the relevant group on your affinity map. Actions might include:
```
Comment (joolswood) [inline content/user-research/analysing-user-research/decide.md]: ok

File: content/user-research/analysing-user-research/determine.md [hunk 15]
```diff
@@ -2,9 +2,9 @@
 layout: text/textblock
 quote: What's the problem that the user is experiencing?
 ---
-The final part of analysis involves reviewing the notes in each group to determine what the observations are telling you.
+The final part of analysis is determining what the observations are telling you.
```
Comment (joolswood) [inline content/user-research/analysing-user-research/determine.md]: Ok

File: content/user-research/analysing-user-research/determine.md [hunk 16]
```diff
@@ -2,9 +2,9 @@
...
 
-When you agree on what you’ve learned, write it as a finding or ‘insight’ on a different coloured sticky note and add it to the relevant group on your affinity map.
+When you agree on what you’ve learned, write it as a finding or ‘insight’ on a different coloured sticky note. Add it to the relevant group on your affinity map.
```
Comment (joolswood) [inline content/user-research/analysing-user-research/determine.md]: Ok

File: content/user-research/analysing-user-research/extract-head.md [hunk 17]
```diff
@@ -1,4 +1,4 @@
 ---
 layout: nav/section
-section: Extract observations
+section: How to extract observations
```
Comment (joolswood) [inline content/user-research/analysing-user-research/extract-head.md]: Ok

File: content/user-research/analysing-user-research/extract.md [hunk 18]
```diff
@@ -3,8 +3,8 @@ layout: text/textblock
 ---
 Follow these steps to gather observations:
 1. Hand out sticky notes.
-2. Ask people to review the notes they took during the research and write down anything interesting or relevant that they saw or heard during the session on the sticky notes. You can use the colour of sticky notes strategically (for example, red to note pain points, green for notes about what works well).
-3. If you have recordings, you can make them available so people can confirm their observations and get verbatim quotes.
+2. Ask team members to review the notes they took during the research. They can then write interesting or relevant observations on the sticky notes. You can use the colour of sticky notes strategically (for example, red to note pain points, green for notes about what works well).
```
Comment (joolswood) [inline content/user-research/analysing-user-research/extract.md]: Ok

File: content/user-research/analysing-user-research/extract.md [hunk 19]
```diff
@@ -3,8 +3,8 @@ layout: text/textblock
...
+3. If you have recordings, you can make them available. Team members can use them to confirm their observations and get verbatim quotes.
```
Comment (joolswood) [inline content/user-research/analysing-user-research/extract.md]: Ok

File: content/user-research/analysing-user-research/index.yml [hunk 20]
```diff
@@ -12,6 +12,8 @@ main:
   - meeting.md
   - plan-head.md
   - plan.md
+  - when-head.md
```
Comment (joolswood) [inline content/user-research/analysing-user-research/index.yml]: Ok

File: content/user-research/analysing-user-research/index.yml [hunk 21]
```diff
@@ -12,6 +12,8 @@ main:
...
+  - when.md
```
Comment (joolswood) [inline content/user-research/analysing-user-research/index.yml]: ok

File: content/user-research/analysing-user-research/intro.md [hunk 22]
```diff
@@ -9,8 +9,8 @@ User research activities produce a lot of raw data, for example:
 - sketches and photos
 - audio and video recordings
 
-You need to filter, organise and interpret this data so you can produce useful and meaningful insights that will help you design and build your service.
+Filtering and organising this data will help you to produce meaningful insights.
```
Comment (joolswood) [inline content/user-research/analysing-user-research/intro.md]: Ok

File: content/user-research/analysing-user-research/intro.md [hunk 23]
```diff
@@ -9,8 +9,8 @@ User research activities produce a lot of raw data, for example:
...
 
-Follow your [user research plan](/user-research/planning-user-research) and keep asking yourself 'What's the problem that the user is experiencing?'
+Follow your [user research plan](/user-research/planning-user-research) and keep asking yourself 'What is the problem that the user is experiencing?'
```
Comment (joolswood) [inline content/user-research/analysing-user-research/intro.md]: Ok

File: content/user-research/analysing-user-research/plan-head.md [hunk 24]
```diff
@@ -1,4 +1,4 @@
 ---
 layout: nav/section
-section: Plan analysis sessions
+section: Who to involve in analysing research
```
Comment (joolswood) [inline content/user-research/analysing-user-research/plan-head.md]: Ok

File: content/user-research/analysing-user-research/plan.md [hunk 25]
```diff
@@ -1,8 +1,6 @@
 ---
 layout: text/textblock
 ---
-Invite anyone who observed the user research to take part in the analysis session. You should do analysis as soon as you can after each round of research, while it is still fresh in people’s minds.
+Invite anyone who observed the user research to take part in analysis sessions.
```
Comment (joolswood) [inline content/user-research/analysing-user-research/plan.md]: Ok

File: content/user-research/analysing-user-research/sections.md [hunk 26]
```diff
@@ -2,8 +2,9 @@
 layout: nav/sections
 sections:
   - Meeting the Digital Service Standard
-  - Plan analysis sessions
-  - Extract observations
+  - Who to involve in analysing research
```
Comment (joolswood) [inline content/user-research/analysing-user-research/sections.md]: Ok

File: content/user-research/analysing-user-research/share.md [hunk 27]
```diff
@@ -2,6 +2,6 @@
 layout: text/textblock
 ---
 
-After analysing your research, collate your findings so you can share them with the wider team and stakeholders (sometimes called a 'shareback'). You can do this on an infinity map, an electronic presentation or whatever medium suits your team.
+Collate your findings so you can share them with the wider team and stakeholders. This is sometimes called a 'shareback'.
```
Comment (joolswood) [inline content/user-research/analysing-user-research/share.md]: ok

File: content/user-research/analysing-user-research/share.md [hunk 28]
```diff
@@ -2,6 +2,6 @@
 layout: text/textblock
 ---
 
-After analysing your research, collate your findings so you can share them with the wider team and stakeholders (sometimes called a 'shareback'). You can do this on an infinity map, an electronic presentation or whatever medium suits your team.
+Collate your findings so you can share them with the wider team and stakeholders. This is sometimes called a 'shareback'. 
 
-You can share insights in different ways. If you've been testing prototypes you might show printouts with comments on post-its. If you've only just started you might be reading out quotes and observations.
+You can share insights in different ways. If you've been testing prototypes you might show printouts with comments on sticky notes. If you've only just started you might read out quotes and observations. You can also use an electronic presentation or whatever medium suits your team.
```
Comment (joolswood) [inline content/user-research/analysing-user-research/share.md]: Ok

File: content/user-research/analysing-user-research/shareback.md [hunk 29]
```diff
@@ -10,4 +10,4 @@ caption: Caption: an example of a user research shareback wall.
 stackPosition: top
 ---
 
-‘A shareback wall is a way to bring back what we learnt from user research to the team. We can use it to quickly see what’s working (green) and where there are problems we still need to solve (in red and orange).’ — User researcher
+‘A shareback wall is a way to bring back what we have learnt from user research to the team. We can use it to quickly see what’s working (green) and where there are problems we still need to solve (in red and orange).’ — User researcher
```
Comment (joolswood) [inline content/user-research/analysing-user-research/shareback.md]: Ok

File: content/user-research/analysing-user-research/sort.md [hunk 30]
```diff
@@ -2,18 +2,18 @@
 layout: text/textblock
 ---
 
-Once they’ve written down their observations, ask your team members to place their sticky notes on a wall and start sorting them into similar themes. You can group by:
+Once team members have written down their observations, ask them to place their sticky notes on a wall. Start sorting them into similar themes. You can group them by:
```
Comment (joolswood) [inline content/user-research/analysing-user-research/sort.md]: Ok

File: content/user-research/analysing-user-research/sort.md [hunk 31]
```diff
@@ -2,18 +2,18 @@
...
 - common topics (for example, identity, delivery, payment)
 - stages in a user journey (for example, ‘supply photo’, ‘attend interview’, ‘pay’)
 - individual pages or steps in a transaction
 - types of user (for example, first time user, business user)
 
-Allow people to move notes placed by other people. The idea is to look for patterns or clusters, and to keep moving the notes until clear groups emerge. This is often called ‘affinity diagramming’ or ‘affinity sorting/mapping’.
+Allow people to move notes placed by other people. The idea is to look for patterns or clusters, and to keep moving the notes until clear groups emerge. This is often called ‘affinity mapping’.
```
Comment (joolswood) [inline content/user-research/analysing-user-research/sort.md]: Ok

File: content/user-research/analysing-user-research/sort.md [hunk 32]
```diff
@@ -2,18 +2,18 @@
...
 
 ### [2]Name your groups
-Once you have your groups, agree a title for each. At this stage, you can discard irrelevant or isolated notes.
+Once you have your groups, agree on a title for each. At this stage, you can discard irrelevant or isolated notes.
```
Comment (joolswood) [inline content/user-research/analysing-user-research/sort.md]: Ok

File: content/user-research/analysing-user-research/sort.md [hunk 33]
```diff
@@ -2,18 +2,18 @@
...
 
-You should also check if you can break any large groups into smaller themes based on matching observations. For example, if users have to supply a photo to use your service, you might have a ‘photos’  group that could broken down into:
+Check if you can break large groups into smaller themes based on matching observations. For example, if users have to supply a photo to use your service, you might have a ‘photos’  group that could broken down into:
```
Comment (joolswood) [inline content/user-research/analysing-user-research/sort.md]: Ok

File: content/user-research/analysing-user-research/when-head.md [hunk 34]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (joolswood) [inline content/user-research/analysing-user-research/when-head.md]: Ok

File: content/user-research/analysing-user-research/when.md [hunk 35]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (joolswood) [inline content/user-research/analysing-user-research/when.md]: Ok

File: content/user-research/analysing-user-research/affinity.md (hunk 12 above)
Comment (shad0wMe0w) [inline content/user-research/analysing-user-research/affinity.md]: This is an attempt to _revert_ what's currently on prod. It should be the same as what's on develop: 'straight away'. @meaghan-n correct?

File: content/user-research/consent-forms/crafting-consent-form.md [hunk 36]
```diff
@@ -2,12 +2,12 @@
 layout: text/textblock
 ---
 
-Every user research session is different. You need to think about all of the kinds of information you may need to capture during the research and adjust the consent form to include them. For example, the participant may be sharing artefacts that contain their address and details of disability.
+Every user research session is different. Think about all the information you may need to capture during the research. Adjust the consent form to reflect this. For example, the participant may be sharing artefacts that contain their address and details of disability.
```
Comment (joolswood) [inline content/user-research/consent-forms/crafting-consent-form.md]: Ok

File: content/user-research/consent-forms/crafting-consent-form.md [hunk 37]
```diff
@@ -2,12 +2,12 @@
...
 
 The most important information you need in a consent form is:
-- who is conducting the research — the researcher, the team and also the agency
+- who is conducting the research — the researcher, team and agency
```
Comment (joolswood) [inline content/user-research/consent-forms/crafting-consent-form.md]: Ok

File: content/user-research/consent-forms/crafting-consent-form.md [hunk 38]
```diff
@@ -2,12 +2,12 @@
...
 - why you are doing the research
-- what things you will be recording
+- what you will be recording
```
Comment (joolswood) [inline content/user-research/consent-forms/crafting-consent-form.md]: Ok

File: content/user-research/consent-forms/crafting-consent-form.md [hunk 39]
```diff
@@ -2,12 +2,12 @@
...
 - what you will do with the information and recording
-- how the user can withdraw their consent (for example, a phone number or email address)
+- how the user can withdraw their consent (including a contact phone number or email address)
```
Comment (joolswood) [inline content/user-research/consent-forms/crafting-consent-form.md]: Ok

File: content/user-research/consent-forms/get-consent-head.md [hunk 40]
```diff
@@ -1,4 +1,4 @@
 ---
 layout: nav/section
-section: Get consent before you research
+section: Get informed consent before you research
```
Comment (joolswood) [inline content/user-research/consent-forms/get-consent-head.md]: Ok

File: content/user-research/consent-forms/get-consent.md [hunk 41]
```diff
@@ -2,8 +2,8 @@
 layout: text/textblock
 ---
 
-Before you can do user research you need to get the consent of the user.
+You need to get the participant's informed consent before you can do user research.
```
Comment (joolswood) [inline content/user-research/consent-forms/get-consent.md]: Ok

File: content/user-research/consent-forms/get-consent.md [hunk 42]
```diff
@@ -2,8 +2,8 @@
...
 
-The process of getting consent usually starts with a conversation before you begin the research session. You explain to the person you’re researching with what the research is about and what it will involve.
+Make sure you have informed consent before you begin the research session. Explain to the participant what the research is about and what it will involve.
```
Comment (joolswood) [inline content/user-research/consent-forms/get-consent.md]: Ok

File: content/user-research/consent-forms/intro.md [hunk 43]
```diff
@@ -4,7 +4,4 @@ category: User research
 title: Consent forms for user research
 subtitle: How to ensure your users are comfortable and feel safe sharing their experiences.
 ---
-
-To build a service that meets user needs you and your team will need to do research with users.
-
-A consent form helps users to understand how the information they share will be used by the government. It also gives them confidence that they will have control over how their information is used.
+A consent form helps users to understand how the government will use the information they share. It also gives them confidence that they will have control over their information.
```
Comment (joolswood) [inline content/user-research/consent-forms/intro.md]: Ok

File: content/user-research/consent-forms/safe-personal-information-head.md [hunk 44]
```diff
@@ -1,4 +1,4 @@
 ---
 layout: nav/section
-section: Keep personal information safe
+section: Keep personal information secure
```
Comment (joolswood) [inline content/user-research/consent-forms/safe-personal-information-head.md]: Ok

File: content/user-research/analysing-user-research/decide.md (hunk 13 above)
Comment (shad0wMe0w) [inline content/user-research/analysing-user-research/decide.md]: This reads more clearly. Not currently on prod. Recommend accepting this change.

File: content/user-research/consent-forms/safe-personal-information.md [hunk 45]
```diff
@@ -2,6 +2,6 @@
 layout: text/textblock
 ---
 
-You need to make sure the information you collect from users is kept safe. There are rules around [how the government can use personal information](https://www.oaic.gov.au/individuals/privacy-fact-sheets/general/privacy-fact-sheet-17-australian-privacy-principles).
+There are rules around [how the government can use personal information](https://www.oaic.gov.au/individuals/privacy-fact-sheets/general/privacy-fact-sheet-17-australian-privacy-principles). You need to make sure the information you collect is secure from misuse or unauthorised access.
```
Comment (joolswood) [inline content/user-research/consent-forms/safe-personal-information.md]: Ok

File: content/user-research/consent-forms/safe-personal-information.md [hunk 46]
```diff
@@ -2,6 +2,6 @@
 layout: text/textblock
 ---
 
-You need to make sure the information you collect from users is kept safe. There are rules around [how the government can use personal information](https://www.oaic.gov.au/individuals/privacy-fact-sheets/general/privacy-fact-sheet-17-australian-privacy-principles).
+There are rules around [how the government can use personal information](https://www.oaic.gov.au/individuals/privacy-fact-sheets/general/privacy-fact-sheet-17-australian-privacy-principles). You need to make sure the information you collect is secure from misuse or unauthorised access. 
 
-Some users may need their information to be treated differently (for example, if the user has a disability or if they have children present during a recording). The user researcher will need to factor these into how they [plan user research](/user-research/planning-user-research/).
+Some users may need their information to be treated differently (for example, if the user has a disability or if they have children present during a recording). Factor this into your [user research plan](/user-research/planning-user-research/).
```
Comment (joolswood) [inline content/user-research/consent-forms/safe-personal-information.md]: Ok

File: content/user-research/consent-forms/sections.md [hunk 47]
```diff
@@ -2,10 +2,10 @@
 layout: nav/sections
 sections:
   - Meeting the Digital Service Standard
-  - Get consent before you research
+  - Get informed consent before you research
```
Comment (joolswood) [inline content/user-research/consent-forms/sections.md]: Ok

File: content/user-research/consent-forms/sections.md [hunk 48]
```diff
@@ -2,10 +2,10 @@
...
   - Crafting a consent form
   - Using a consent form in an interview
   - Handle receipts for incentives and consent forms separately
-  - Keep personal information safe
+  - Keep personal information secure
```
Comment (joolswood) [inline content/user-research/consent-forms/sections.md]: Ok

File: content/user-research/consent-forms/using-consent-form.md [hunk 49]
```diff
@@ -2,14 +2,14 @@
...
+Explain that they have a choice about what they wish to consent to. Get permission before starting any form of recording (audio, visual or written).
 
-After the interview you should make sure the user is still comfortable with letting you use the information. Sometimes a user may say something during the interview that they don’t feel comfortable sharing.
+Sometimes a participant may say something that they don’t feel comfortable sharing. After the interview, make sure they still consent to you using the information. 
 
 The DTA has a blog about [how to ensure informed consent in user research.](https://www.dta.gov.au/blog/informed-consent-in-user-research/)
 
-You should leave a copy of the consent form with the participant at the end of the session so they have a record of what they have agreed to and know how they can withdraw consent if they want to later.
+Leave a copy of the consent form with the participant at the end of the session. This gives them a record of what they have agreed to. It also lets them know how they can withdraw consent if they want to later.
```
Comment (joolswood) [inline content/user-research/consent-forms/using-consent-form.md]: Ok

File: content/user-research/consent-forms/using-consent-form.md [hunk 50]
```diff
@@ -2,14 +2,14 @@
 layout: text/textblock
 ---
 
-You should start [interviews with users](/user-research/interviewing-users) by explaining the purpose of the research and showing them the consent form.
+You should start [interviews with users](/user-research/interviewing-users) by explaining the purpose of the research. Show participants the consent form.
```
Comment (joolswood) [inline content/user-research/consent-forms/using-consent-form.md]: Ok

File: content/user-research/consent-forms/using-consent-form.md [hunk 51]
```diff
@@ -2,14 +2,14 @@
...
 
-You should explain to the participant that they have a choice about what they wish to consent to, and get permission before starting any form of recording (audio, visual and written).
+Explain that they have a choice about what they wish to consent to. Get permission before starting any form of recording (audio, visual or written).
```
Comment (joolswood) [inline content/user-research/consent-forms/using-consent-form.md]: Ok

File: content/user-research/consent-forms/using-consent-form.md [hunk 52]
```diff
@@ -2,14 +2,14 @@
...
 
-After the interview you should make sure the user is still comfortable with letting you use the information. Sometimes a user may say something during the interview that they don’t feel comfortable sharing.
+Sometimes a participant may say something that they don’t feel comfortable sharing. After the interview, make sure they still consent to you using the information.
```
Comment (joolswood) [inline content/user-research/consent-forms/using-consent-form.md]: Ok

File: content/user-research/find-user-research-participants/bias.md [hunk 53]
```diff
@@ -2,13 +2,13 @@
 layout: text/textblock
 ---
 
-It’s hard to recruit an unbiased sample of user research participants. This is because you’re likely to include some people and exclude others depending on:
+There is a risk that groups of people can be included or excluded, depending on:
```
Comment (shad0wMe0w) [inline content/user-research/find-user-research-participants/bias.md]: Ok

File: content/user-research/find-user-research-participants/bias.md [hunk 54]
```diff
@@ -2,13 +2,13 @@
...
 - what the activity is
-- when the sessions are scheduled
-- where you’re doing the research
+- when you schedule the sessions 
+- where you do the research
 - the method you use to recruit your users
 
-The best way to limit this risk is to use a variety of user research activities and recruitment approaches. For example, in Discovery stage don't just rely on interviews. Do some contextual research with people who are using the service and lab research with users of existing services.
+Limit this risk by using a variety of research activities and recruitment approaches. For example, don't just rely on interviews in Discovery stage. Do some contextual research with users of existing services.
```
Comment (shad0wMe0w) [inline content/user-research/find-user-research-participants/bias.md]: Ok

File: content/user-research/find-user-research-participants/brief-ctd.md [hunk 55]
```diff
@@ -1,8 +1,10 @@
 
 Send the agency a written brief, even if they’re happy to take instructions over the phone. This will be a record in case there are problems with the recruitment.
 
-### [2]Review the screener
+### [2]Review the participant screener
 
-The agency will send you a screener based on the criteria you sent them.
+The recruitment brief usually forms the basis of a recruitment ‘screener’. This is used by the agency to interview potential participants. It helps them to ensure you have the correct mix of people for your research.
```
Comment (shad0wMe0w) [inline content/user-research/find-user-research-participants/brief-ctd.md]: Ok to pull

File: content/user-research/find-user-research-participants/brief.md [hunk 56]
```diff
@@ -2,21 +2,16 @@
...
 In your brief, you should cover:
-- research dates, including times and length of each session, the break you need between sessions, and the number of sessions you wish to do in a single day
+- research dates
+- times and lengths of each session
+- breaks between sessions
+- the number of sessions you wish to do on each date
 - research location
 - the number of participants you want to recruit
-- a description of the types of people you want to recruit (often referred to by agencies as ‘recruitment criteria’)
+- a description of the people you want to recruit (often referred to by agencies as ‘recruitment criteria’)
```
Comment (shad0wMe0w) [inline content/user-research/find-user-research-participants/brief.md]: Ok

File: content/user-research/find-user-research-participants/finding.md [hunk 57]
```diff
@@ -9,6 +9,8 @@ There are different ways to recruit participants:
...
 - invite existing users of your service to take part
 
-Agencies are generally best for recruiting end users. Recruiting participants is quite a lot of work and agencies tend to be able to do this at a reasonable rate. This can be good value for money. For specific user groups, a relevant professional body, charity or community group might be more effective.
+It's generally best to use a research recruitment agency for recruiting end users. Recruiting participants is a lot of work and agencies tend to be able to do it at a reasonable rate. This can be good value for money. 
 
-You will need to think about and budget for [paying incentives](/user-research/paying-incentives/) to the users who participate in research. Research Recruitment Agencies can handle incentive payments for you as a part of their services.
+But you may also need to try different ways of finding specific users. Relevant professional bodies, charities or community groups might be more effective.
+
+Think about and budget for [paying incentives](/user-research/paying-incentives/) to research participants. Research recruitment agencies can handle incentive payments.
```
Comment (shad0wMe0w) [inline content/user-research/find-user-research-participants/finding.md]: Ok

File: content/user-research/find-user-research-participants/intro.md [hunk 58]
```diff
@@ -4,12 +4,13 @@ subtitle: How to recruit current or future potential users of the service.
...
  - are disabled or use assistive technologies
- - have low digital skills or poor literacy
- - come from cultures with different experiences of government
+ - have low digital skills
+ - have low literacy
+ - come from culturally and linguistically diverse backgrounds
  - may need help to use your service
 
-Make sure this is documented in your [user research plan](/user-research/planning-user-research/).
+Document user groups in your [user research plan](/user-research/planning-user-research/).
```
Comment (shad0wMe0w) [inline content/user-research/find-user-research-participants/intro.md]: Ok

File: content/content-strategy/removing-content/archive-record-content/archiving-comm02.md (hunk 1 above)
Comment (shad0wMe0w) [reply content/content-strategy/removing-content/archive-record-content/archiving-comm02.md]: @joolswood would you mind editing this then here? So we can approve all?

File: content/user-research/analysing-user-research/affinity.md (hunk 12 above)
Comment (shad0wMe0w) [reply content/user-research/analysing-user-research/affinity.md]: Ok

**Comment by joolswood** [comment]
@dominikwilkowski, @shad0wMe0w and I have reviewed and content seems good to merge.

**Pull request closed without merging**